robot-mode.el

```diff
@@ -329,10 +245,14 @@ Prefix the continuation with indentation, ellipsis and spacing."
 
 \\{robot-mode-map}"
 
-  (setq-local indent-line-function #'robot-mode-indent-line)
+  (setq-local indent-line-function #'robot-indent-line-function)
   (setq-local font-lock-defaults '(robot-mode-font-lock-keywords nil t))
   (setq-local comment-start "#")
   (setq-local comment-start-skip "#+ *")
+
+  ;; Don't reindent automatically, because keyword and defun name are similar
+  (setq-local electric-indent-inhibit t)
+
```

Comment:
This is shame, but I haven't found a way to avoid it.

Because of the possibles indentation level between the Test name and a keyword. We need to disable the auto indentation occuring when pressing `RETURN`.

Note that this is only when the indentation level is 0 or 1. Maybe a custom electric indent function for robot that only inihit it when at those levels.